src/main/java/io/stargate/sgv2/jsonapi/service/operation/reranking/Feature.java

```diff
@@ -0,0 +1,43 @@
+package io.stargate.sgv2.jsonapi.service.operation.reranking;
```

Comment:
Is this specific to reranking? If not, should probably be in some other package. But if it is, maybe name should reflect it (see another comment below)

Reply:
+1 for it being general, could be in the metrics package